src/pydicom/dataelem.py

```diff
@@ -8,8 +8,11 @@
 """
 
 import base64
+from dataclasses import dataclass
+from io import BufferedIOBase
 import json
-from typing import Optional, Any, TYPE_CHECKING, NamedTuple
+from typing import Optional, Any, TYPE_CHECKING, NamedTuple, cast
+from collections.abc import Iterator
```

Comment:
Add `Iterator` import to the next line instead?

Reply:
Actually wasn't being used - y'all may want to add a check for unused imports.